internal/apischema/awsbedrock/awsbedrock.go

```diff
@@ -458,10 +458,10 @@ type ToolSpecification struct {
 	// The description for the tool.
 	Description *string `json:"description,omitempty"`
 
-	// The input schema for the tool in JSON format.
+	// The schema for the tool in JSON format.
 	//
-	// InputSchema is a required field
-	InputSchema *ToolInputSchema `json:"inputSchema"`
+	// Schema is a required field
+	Schema *ToolSchema `json:"schema"`
```

Comment:
This should not be changed, it is the AWS bedrock schema

Reply:
offline discussion - fixed this. didn't realize this was a field required by aws and that their are issues using their api for these types.

Reply:
note: I think we should have a test for if our hard coded fields and theirs misalign